mapping.csv

```diff
@@ -170,15 +170,14 @@
 "^simda$",simda,malpedia,https://www.us-cert.gov/ncas/alerts/TA15-105A
 "^skeeyah$",skeeyah,
 "^skunkx$",skunkx,
-"^smokeloader$",smokeloader,malepdia
+"^smokeloader$",smokeloader,malpedia
 "^snappeabot$",snappea,
 "^sniperspy$",sniperspy,
 "^sprotect$",sprotect,
 "^spyeye$",spyeye,
 "^stantinko$",stantinko,malpedia
 "^(sykipot(-apt)?|getkys)$",sykipot,malpedia
-"^tdss$",tdss,
-"^techsnab$",techsnab,
+"^tdss$",tdss,alureon,https://en.wikipedia.org/wiki/Alureon
```

Comment:
Is this a mistake and did you want to map tdss to alureon (would be my first preferred name, as it's the name of the wikipedia page, i.e. more often used)

Reply:
Not a mistake. I think TDSS is more commonly used within analysts.